joplin/base/wagtail_hooks.py

```diff
@@ -300,6 +301,158 @@ def register_link_handler(features):
     features.register_link_type(InternalLinkHandler)
 
 
+# In here we're going to remake all the urls for this page
+@hooks.register('after_edit_page')
+def after_edit_page(request, page):
+    # Make sure we're publishing so our Janis urls reflect the live site
+    # taking this from https://github.com/wagtail/wagtail/blob/349fca66f32ecf3df6316b0cffb4e5fd0d24d6c7/wagtail/admin/views/pages.py#L228
+    publishing = bool(request.POST.get('action-publish'))
+    if not publishing:
+        return
+
+    # Clear out all old urls
+    for janis_url in page.janis_urls.all():
+        janis_url.janis_url.delete()
+
+    # Create the new urls
+    new_urls = []
+
+    # Page type matters here
+    page_type = page._meta.object_name
+
+    # todo: not have magic language arrays hiding around the codebase
+    languages = ['en', 'es']
```

Comment:
you can get these from settings.LANGUAGES

Reply:
thought about that, this is one of the places that the disconnect between Janis and Joplin comes into play, we only have english and spanish in Janis, so it doesn't make sense to have /vi/ and /ar/ urls in here, even though we do have the ability to enter that content in Joplin.